src/background.js

```diff
@@ -133,7 +133,7 @@ const messageOps = {
     delete message.tabId;
     toSave[idForItem(message)] = message;
     return browser.tabs.query({}).then(tabs => {
-      if (tabs.length <= 1) {
+      if (tabs.length < 1) {
```

Comment:
I'd be happy with the `if ((tabId && tabs.length <= 1) || tabs.length < 1)` fix.  Heck, we could even make it `if ((tabId && tabs.length === 1) || tabs.length < 1)`, to be extra clear that we're closing the last tab…  Although, that's kinda weird, cause how would we even be clicking on a button in a window with no tabs?

Maybe the real check should be `if (tabId && tabs.length <= 1)`.